### ISSUE TITLE
Laptop Freezing While Training 

### ISSUE THREAD

**Issue by Faheemibrahim** [ultralytics/ultralytics#7499]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

i have and 4GB GTX Nvidia 1050 and my laptop freezes while training. is this hardware limitations ? or can i add other parameters to avoid the laptop from freezing. I am running the code on visual studio on ubuntu 20,04 and the code below is what i am using for training. i have tried cache=true and it was the same issue. i have played around with the batch size. i haven't tired training the model on cpu for long. is it possible to use both cpu and gpu for training the dataset ? if yes how can i implement it in the code.also while training the MAP value does not show up.

from ultralytics import YOLO

model = YOLO('yolov8n.pt')

results = model.train(data='/home/faheem/VENVS/cv/pythoncv/Rock-Paper-Scissors-SXSW-11/data.yaml',batch = 1, epochs=10, imgsz=320,device = '0')

**Comment by Faheemibrahim** [comment]
solved !! -> https://github.com/ultralytics/ultralytics/issues/4106#issuecomment-1887419697

**Comment by glenn-jocher** [comment]
@Faheemibrahim i'm glad to hear you've resolved the issue with the mAP value not displaying during training. For future reference and for others who might encounter a similar issue, the solution can be found in the discussion linked in your comment. Remember, the mAP metric is a key indicator of model performance, and it's important to monitor it to understand how well your model is learning to detect objects in your dataset. If you have any more questions or need further assistance, feel free to reach out. Happy training! 🚀
